configmaps/otel-collector-monarch.yaml

```diff
@@ -0,0 +1,165 @@
+# Copyright 2024 Google LLC
```

Comment:
A bit weird to create a configmaps/ folder just for this file.

Reply:
Was putting it as a parallel with crds, let me know if there is a better location.

Reply:
Or maybe add it in Config Sync repo?

Reply:
Yeah, adding it in the CS repo seems better